framework/YapDatabaseExtensions/Common/Remove.swift

```diff
@@ -26,7 +26,7 @@ extension YapDatabaseReadWriteTransaction {
     :param: indexes An Array<YapDB.Index>
     */
     public func removeAtIndexes(indexes: [YapDB.Index]) {
-        indexes.map(removeAtIndex)
+        _ = indexes.map(removeAtIndex)
```

Comment:
Also, sorry to be annoying, but can you re-direct this PR to `development`? Sorry - I'm updating the default branch now.